src/hevm/src/EVM.hs

```diff
@@ -1911,8 +1938,12 @@ create self this xGas xValue xs newAddr initCode = do
         touchAccount self
         touchAccount newAddr
         let
+          store = case view (env . storageModel) vm0 of
+            ConcreteS -> Concrete mempty
+            SymbolicS -> Symbolic $ sListArray (Just 0) []
```

Comment:
```suggestion
            SymbolicS -> Symbolic $ sListArray Nothing []
```

Reply:
This is creating a new contract, so it's storage is assumed to be empty. We keep storage fully symbolic only if the account has already been initialized previous to this transaction; i.e. when it's loaded from rpc, is the initial contract, or loaded from state (currently not yet supported)

Reply:
ah had missed that this was in `create` :+1: